CMakeLists.txt

```diff
@@ -500,7 +500,7 @@ if(ENABLE_CONAN)
     "lua/${CONAN_LUA_VERSION}"
     "onetbb/${CONAN_TBB_VERSION}"
     BASIC_SETUP 
-    GENERATORS cmake_find_package
+    GENERATORS cmake_find_package json
```

Comment:
It would just generate it in build folder, right? May be add brief comment why do we need this `json`? I mean someone in the future may just delete it because it is not used in this repo itself.

Reply:
> It will just generate it in build folder, right?

Right. Added a comment why it's useful in the source.